mar/lambda_function.py

```diff
@@ -7,7 +7,23 @@
 import subprocess
 
 class Context(object):
+    """Context provides runtime information to handler"""
```

Comment:
Your documentation should convey What is this context useful for ?

Runtime Information to which handler ?

Reply:
Modified accordingly